README.md

```diff
@@ -1,17 +1,92 @@
 
 # Todo
 
-Sample application
+Todo backend example for [Todo Backend](http://www.todobackend.com/index.html) using Hexagon. Built 
+with [Travis CI](https://travis-ci.org) and hosted at Heroku here: 
+[https://todo-backend-hexagonkt.herokuapp.com/](https://todo-backend-hexagonkt.herokuapp.com/)
 
-# Modules
+The spec is:
 
-## Backend
+GET `/tasks/{id}`
 
-Depends on the frontend and includes its output to the resources in order to serve The main page
-with the Frontend JS.
+Response:
+```json
+{
+    "url": "/tasks/{id}",
+    "title": string
```

Comment:
Some code blocks are not valid JSON. Not a very important issue though.

Reply:
I removed the json specification since you should be able to see the types of the values